lib/wallet/wallet.js

```diff
@@ -1795,7 +1799,18 @@ class Wallet extends EventEmitter {
   async sendBid(name, value, lockup, options) {
     const unlock = await this.fundLock.lock();
     try {
-      return await this._sendBid(name, value, lockup, options);
+      const idempotencyKey = options ? options.idempotencyKey : null;
+      if (idempotencyKey) {
+        if (this.sendBidResults.has(idempotencyKey)) {
+          return this.sendBidResults.get(idempotencyKey);;
```

Comment:
semicolon; does your editor running this repo's eslint?

Reply:
No, I'll set that up.